venus-worker/src/sealing/worker/task/planner/snapup.rs

```diff
@@ -118,7 +118,7 @@ impl<'c, 't> SnapUp<'c, 't> {
 
         let allocated = match maybe_allocated {
             Some(a) => a,
-            None => return Ok(Event::Retry),
+            None => return Ok(Event::Idle),
```

Comment:
`Event::Idle` should cover  `maybe_allocated` ?

Reply:
resolved.